rex/renewable_resource.py

```diff
@@ -600,7 +714,14 @@ def _get_ds(self, ds_name, ds_slice):
             ts2 = super()._get_ds('{}_{}m'.format(var_name, h2), ds_slice)
 
             if (var_name == 'windspeed') and extrapolate:
-                out = self.power_law_interp(ts1, h1, ts2, h2, h)
+                if h < h1:
+                    try:
+                        self._try_monin_obukhov_extrapolation(ts1, ds_slice,
```

Comment:
Wait so what does this do? monin is used to perform extrapolation for other variables?

Reply:
This is a surface level extrapolation scheme that uses the Moninobuhkov length and the roughness length of the air (z0).  We don't have z0 in the WTK data but will in WTKv2.  Mike used this to extrapolate below 10m

Reply:
So it will do normal interpolation/extrapolation above the lowest height but this fancy extrapolation below the lowest height? I'm guessing its a boundary layer surface friction thing?

Reply:
It will try to run the monin_obuhkov_extrapolation below 10m, but if it doesn't have the required inputs it will revert to power_law extrapolation

Yeah it is a boundary layer correction scheme